examples/SampleUtils.cpp

```diff
@@ -81,6 +81,75 @@ namespace utils {
         return prod;
     }
 
+    const ml::Operator createActivationOperator(std::vector<SHARED_DATA_TYPE>& constants,
+                                                const ml::GraphBuilder& builder,
+                                                FusedActivation activation,
+                                                const std::vector<float>& options) {
+        ml::Operator activationOperator;
+        if (activation == FusedActivation::RELU) {
+            activationOperator = builder.ReluOperator();
+        } else if (activation == FusedActivation::RELU6) {
+            ml::ClampOptions clampOptions;
+            float min = 0;
+            auto minConstant = std::make_shared<std::vector<char>>(sizeof(float));
```

Comment:
`
Could it be constructed directly?
```c++
std::vector<float> minConstant = {0};
std::vector<float> maxConstant = {6};
```

Reply:
I use std::make_shared<std::vector<char>> here to avoid memory releasing too early.

Reply:
I got it. Then how about use `SHARED_DATA_TYPE` to be consistent with `constants` and use assignment instead of `memcpy`. such as
```c++
auto minConstant = SHARED_DATA_TYPE(new std::vector<char>(sizeof(float)));
*(reinterpret_cast<float*>(minConstant->data())) = 0;
constants.push_back(minConstant);
```

Reply:
Thanks! I will update as your suggestions.